packages/client/src/pages/Tutorial/Tutorial.tsx

```diff
@@ -21,7 +21,7 @@ const Tutorial = () => {
   return (
     <OuterContainer>
       <InnerContainer>
-        {slideNumber < 4 ? <StyedH1>Story:</StyedH1> : <StyedH1>Tutorial:</StyedH1>}
+        {slideNumber < 4 ? <StyledH1>Story:</StyledH1> : <StyledH1>Tutorial:</StyledH1>}
```

Comment:
На твое усмотрение, но на мой взгляд так быть может лучше читалось бы
```suggestion
        <StyledH1>{slideNumber < 4 ? "Story" : "Tutorial" }</StyledH1>
```